src/test/shell/bazel/bazel_java_test_defaults.sh

```diff
@@ -17,8 +17,6 @@
 # Tests the java rules with the default values provided by Bazel.
 #
```

Comment:
Just noted that this test doesn't have set -euo pipefail

This way we don't notice intermediate command failues (e.g.:

bazel run java/main:JavaBinary --java_toolchain=//java/main:default_toolchain \
      --javabase=@bazel_tools//tools/jdk:remote_jdk11 \
      --verbose_failures -s &>"${TEST_log}"

If this command fails, the test will continue running). Can you pls address this in a separate cl? Thank you so much, sorry for not noticing earlier.

Reply:
Thanks for noticing! I added it in this PR since it's a small change. I'll import it now.